src/index.js

```diff
@@ -0,0 +1,40 @@
+import createScore from '../modules/createScore.js';
+import isNumeric from '../modules/isNumeric.js';
+import setToLocalStorage from '../modules/localStorage.js';
+import './style.css';
+// import '@fortawesome/fontawesome-free/css/all.css';
```

Comment:
- [ ] Do not commit old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history. 📓